kotlin/workflow-core/src/main/java/com/squareup/workflow/WorkflowContext.kt

```diff
@@ -101,18 +101,21 @@ interface WorkflowContext<StateT : Any, in OutputT : Any> {
 
   /**
    * Ensures [child] is running as a child of this workflow, and returns the result of its
-   * [compose][Workflow.compose] method.
+   * `compose` method.
    *
-   * **Never call [Workflow.compose] directly, always do it through this context method.**
+   * **Never call [StatefulWorkflow.compose] or [StatelessWorkflow.compose] directly, always do it
+   * through this context method.**
    *
    * 1. If the child _wasn't_ already running, it will be started either from
    *    [initialState][Workflow.initialState] or its snapshot.
```

Comment:
nit: no longer really an _or_ situation, since we unified the methods.

Reply:
Those are still related though, and I want to avoid confusion about people trying to call them directly.